c2corg_api/scripts/migration/documents/waypoints/summit.py

```diff
@@ -70,6 +72,13 @@ def get_document_locale(self, document_in, version):
             summary=summary
         )
 
+    def get_climbing_outdoor_types(self, document_in):
+        outdoor_types = []
+        if document_in.summit_type == 4:
+            outdoor_types.append('multi')
+        outdoor_types = outdoor_types if outdoor_types else None
+        return outdoor_types
```

Comment:
This function is a bit weird. Why making ``outdoor_types`` a list if you make it ``None`` when empty. I would suggest:
```
return ['multi'] if document_in.summit_type == 4 else None
```
By the way do you really need a function to do this?